src/Compute/Compute/VirtualMachine/Operation/NewAzureVMCommand.cs

```diff
@@ -50,10 +50,11 @@
 using SM = Microsoft.Azure.PowerShell.Cmdlets.Compute.Helpers.Storage.Models;
 using Microsoft.Azure.Commands.Compute;
 using Microsoft.Azure.PowerShell.Cmdlets.Compute.Helpers.Network.Models;
-
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.Compute
 {
+    [GenericBreakingChange("It is recommended to use parameter \"-PublicIpSku Standard\" in order to create a new VM with a Standard public IP.Specifying zone(s) using the \"-Zone\" parameter will also result in a Standard public IP.If \"-Zone\" is not specified, the VM will be created with a Basic public IP instead.Please note that the Standard SKU IPs will become the default behavior for VM creation in the future")]
```

Comment:
Spaces after periods

Reply:
currently: If \"-Zone\" is not specified, the VM will be created with a Basic public IP instead.
I think we should specify:
"If zone AND PublicIPSku are not specificed,"

Reply:
I copied this warning (provided by Annie) from the Work Item. Should I still modify it or keep it as it was given by Annie?

Reply:
Never mind, "If zone AND PublicIPSku are not specificed," that part makes the warning sounds clearer to users. Good catch!